### PR TITLE
Add downtime webform

### PR DESCRIPTION
**Pull request by matyasselmeci**
This is a web form for generating YAML files that an admin can copy-and-paste into a downtime file to report downtime. It is running on topology-itb at https://topology-itb.opensciencegrid.org/generate_downtime.

### CODE REVIEW

**Comment by brianhlin** [comment]
Marking this as `do-not-merge` because we will tell our consumers the day we plan on making this change and should not merge this until then.

**Comment by matyasselmeci** [comment]
Dang, that was fast.

In the meantime, feel free to give it to someone else for review since you probably don't have the time.

File: src/templates/homepage.html.j2 [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{% extends "base.html.j2" %}
+{% block title -%}
+OSG Topology Interface
+{%- endblock %}
+{% block content %}
+<div class=container>
```
Comment (djw8605) [inline src/templates/homepage.html.j2]: `container` should be in quotes.

**Comment by matyasselmeci** [comment]
I don't know enough JavaScript to do that properly, and what I have now took long enough anyway. WTForms does a good job of validating and providing useful feedback already.

**Pull request merged**